src/App.js

```diff
@@ -1,25 +1,81 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+let DATA = [
+  {
+    id: 1,
+    title: "Lorem ipsum dolor sit amet consectetur",
+  },
+  {
+    id: 2,
+    title: "Lorem ipsum dolor sit amet consectetur",
+  },
+];
+function ListTodos({ title, id, deleteTodo }) {
+  const [isCheck, setIsCheck] = React.useState(false);
+  return (
+    <div className="flex justify-between items-center py-3 border-b">
+      <div className="flex gap-10 items-center">
+        <input type="checkbox" onClick={() => setIsCheck(!isCheck)} />
+        {isCheck ? <p className="line-through">{title}</p> : <p>{title}</p>}
+      </div>
+      <button
+        className="text-red-700 bg-slate-200 rounded-full w-10 h-10 flex justify-center items-center text-base"
+        onClick={() => deleteTodo(id)}
+      >
+        <FontAwesomeIcon icon={faTrash} />
+      </button>
+    </div>
+  );
+}
 
-function App() {
+export default function Todos() {
+  const [data, setData] = React.useState(DATA);
+  const [inputValue, setInputValue] = React.useState();
+  function addNewTodo(title) {
```

Comment:
Instead of repeating `React.` everywhere it would be better if you destructure it from the the imports